.gitignore

```diff
@@ -7,4 +7,4 @@ node_modules/
 /temp
 .idea
 /scripts/check-size-out.md
-
+/playground/p/**/package-lock.json
```

Comment:
Can we add a folder here for playgrounds to be auto-ignored and call it out in the readme?

e.g. `/playground/scratch/**`

Reply:
Sure, that's a good idea!

Reply:
Added!